kubeflow/gcp/basic-auth-ingress.libsonnet

```diff
@@ -430,6 +436,45 @@
     ),
     cloudEndpoint:: cloudEndpoint,
 
+    local istioMappingSvc = {
+      apiVersion: "v1",
+      kind: "Service",
+      metadata: {
+        labels: {
+          app: "istioMappingSvc",
+        },
+        name: "istio-mapping-service",
+        namespace: namespace,
+        annotations: {
+          "getambassador.io/config":
+            std.join("\n", [
+              "---",
+              "apiVersion: ambassador/v0",
+              "kind:  Mapping",
+              "name: istio-mapping",
+              "prefix_regex: true",
+              "prefix: /(?!whoami|kflogin).*",
```

Comment:
So this means whoami or kflogin will both be directed to istio-ingressgateway?
Why do we care about whoami here?
Isn't why kflogin goes to istio-ingressgateway?

Reply:
this means anything except /whoami and /kflogin will match.

Reply:
ah I see, thanks